docs/conf.py

```diff
@@ -0,0 +1,40 @@
+# Configuration file for the Sphinx documentation builder.
+
+# -- Project information
+
+project = 'Lumache'
```

Comment:
these should be adjusted.

Reply:
ty!

Reply:
I've edited and regenerated the docs, though you may need to clear cache to see the difference